lib/commands/add.js

```diff
@@ -1,16 +1,25 @@
 "use strict";
 
 const defaultGenerator = require("../generators/add-generator");
-const modifyHelper = require("../utils/modify-config-helper");
+const modifyConfigHelper = require("../utils/modify-config-helper");
 
 /**
  * Is called and returns a scaffolding instance, adding properties
  *
- * @returns {Function} modifyHelper - A helper function that uses the action
+ * @param	{String}	configFile - Name of the existing/default webpack configuration file
+ *
+ * @returns {Function}	modifyConfigHelper - A helper function that uses the action
  * 	we're given on a generator
  *
  */
 
-module.exports = function add() {
-	return modifyHelper("add", defaultGenerator);
+const DEFAULT_WEBPACK_CONFIG_FILENAME = "webpack.config.js";
+
+module.exports = function add(...args) {
+	const filePaths = args.slice(3);
+	let configFile = DEFAULT_WEBPACK_CONFIG_FILENAME;
```

Comment:
No need for outer scope constant here

Reply:
@ematipico [suggested](https://github.com/webpack/webpack-cli/pull/352#discussion_r175936671) to keep a constant. I'm convinced either way. :sweat_smile:

Reply:
store the constant within the scope of the function

Reply:
Fixed.